### ISSUE TITLE
function default_version in py3versions.py fails if os.readlink('/usr/bin/python3') returns a link

### ISSUE THREAD

**Issue by gneiss15** [genuine-linux/python3-defaults#1]
When upgrading python3-twisted (ubuntu 18.04) I got the error:
ValueError: the symlink /usr/bin/python3 does not point to the python3 default version. It must be reset to point to python3.6

This is, because I had installed alternatives to python3.6.
When doing this the symlink '/usr/bin/python3' points to '/etc/alternatives/python3' which in turn points to '/usr/bin/python3.6'.

So the code of the function 'default_version' is (at least) incomplete.

My sugestion:
Insert the following code before the comment '# consistency check' (Line 146):
```
while os.path.islink(_default_version):
    _default_version = os.readlink(_default_version)
```

**Comment by lorenzogrv** [comment]
Hey @gneiss15 , you are probably reporting at wrong repository

**Comment by gneiss15** [comment]
@lorenzogrv Yes, maybe, but what's the right one?

**Comment by lorenzogrv** [comment]
> @lorenzogrv Yes, maybe, but what's the right one?

Maybe at the twisted repo → https://github.com/twisted/twisted

**Comment by gneiss15** [comment]
I don't think that is the right repro. The problem occurs when updating twisted, but twisted uses pythons function default version and this is part of this repro.
